dist/init/linux-sysvinit/README.md

```diff
@@ -0,0 +1,11 @@
+SysVinit conf for Caddy
+=======================
+
+Usage
+-----
+
+* Download the appropriate Caddy binary in `/usr/local/bin/caddy` or use `curl https://getcaddy.com | bash`.
+* Save the SysVinit config file in `/etc/init.d/caddy`.
+* Ensure that the folder `/etc/caddy` exists and that the subfolder `ssl` is owned by `www-data`.
+* Create a Caddyfile in `/etc/caddy/Caddyfile`
+* Now you can use `sudo /etc/init.d/caddy start|stop|restart|reload|status`.
```

Comment:
This `sudo` command is an anti-pattern that will usually result in all sorts of pain at some point.  It is much better to use the `/usr/sbin/service` command, which will make sure that environment variables, ulimits, etc, are changed and done as the system itself would make it happen during startup, instead of whatever current directory, environments variables sudo allows, and so on, to be part of Caddy's invocation.